cmake.deps/CMakeLists.txt

```diff
@@ -63,11 +63,9 @@ endif()
 
 option(USE_EXISTING_SRC_DIR "Skip download of deps sources in case of existing source directory." OFF)
 
-if(WIN32)
-  find_package(Git)
-  if(NOT Git_FOUND)
-    message(FATAL_ERROR "Git is required to apply patches for Windows.")
-  endif()
+find_package(Git)
+if(NOT Git_FOUND)
+  message(FATAL_ERROR "Git is required to apply patches.")
 endif()
```

Comment:
```suggestion
find_package(Git REQUIRED)
```

Reply:
When the libuv patch is merged, we won't need to patch.